resources/translations/fi.edn

```diff
@@ -312,14 +318,16 @@
           :member-added {:subject "Lisätty hakemuksen %3 jäseneksi"
                          :message "Hyvä %1,\n\nSinut on lisätty käyttäjän %4 jättämän hakemuksen %3 jäseneksi.\n\nVoit tarkastella hakemusta ja hyväksyä käyttöehdot osoitteessa %6"}
           :member-invited {:subject "Kutsu osallistumaan hakemukseen %3"
-                           :message "Hyvä %1,\n\nSinut on kutsuttu osallistumaan käyttäjän %2 jättämään käyttöoikeushakemukseen %3.\n\nVoit tarkastella hakemusta ja hyväksyä käyttöehdot osoitteessa %4"}
+                           :message "Hyvä %1,\n\nSinut on kutsuttu osallistumaan käyttäjän %2 jättämään hakemukseen %3.\n\nVoit tarkastella hakemusta ja hyväksyä käyttöehdot osoitteessa %4"}
```

Comment:
Typo still here

Reply:
here it's correct, "Sinut on kutsuttu osallistumaan - - hakemukseen."